public/partials/buyer_new.html

```diff
@@ -1,62 +1,58 @@
+                    </div>
+                </div>
+            </md-card-content>
+            <md-card-content>
 
-            <div layout="row">
-                <div class="profilePicture" flex="30"></div>
-                <div layout="column" flex="70">
-                    <md-input-container flex>
-                        <label>Buyer Name/Company Name</label><input type="text" ng-model="buyer.bu_buyer_name" />
+                <div id="contact">
+                    <md-input-container>
+                        <label>Address 1: </label><input type="text" ng-model="buyer.ad_address.ad_address1"/>
+                    </md-input-container>
+                    <div layout="row">
+                        <md-input-container flex>
+                            <label>Address 2: </label><input type="text" ng-model="buyer.ad_address.ad_address2"/>
+                        </md-input-container>
+
+                        <md-select flex ng-init="buyer.ad_address.pa_parish" placeholder="Parish" ng-model="buyer.ad_address.pa_parish">
+                            <md-option ng-value="parish.pa_parish_name" ng-repeat="parish in parishes">{{parish.pa_parish_name}} | {{parish.pa_parish_code}}</md-option>
+                        </md-select>
+                    </div>
+
+                    <md-input-container>
+                        <label>Cell #: </label><input type="tel" ng-model="buyer.bu_phone"/>
+                    </md-input-container>
+                    <md-input-container>
+                        <label>Email: </label><input type="email" ng-model="buyer.bu_email"/>
                     </md-input-container>
-                    <md-select placeholder="Type" ng-model="buyer.bt_buyer_type._id">
-                        <md-option ng-value="buyerType._id" ng-repeat="buyerType in buyerTypes">{{buyerType.bt_buyer_type_name}}</md-option>
-                    </md-select>
                 </div>
-            </div>
-        </md-card-content>
-        <md-card-content>
 
-        <div id="contact">
-            <md-input-container>
-                <label>Address 1: </label><input type="text" ng-model="buyer.ad_address.ad_address1"/>
-            </md-input-container>
-            <div layout="row">
-                <md-input-container flex>
-                    <label>Address 2: </label><input type="text" ng-model="buyer.ad_address.ad_address2"/>
+                <div id="paymentTerms">
+                    <md-input-container>
+                        <label>Preferred Payment Terms: </label><input type="text" ng-model="buyer.bu_payment_terms"/>
+                    </md-input-container>
+                </div>
+                <md-input-container layout="row" layout-sm="column" layout-align="center center">
+                    <md-button ng-click="save()">Save</md-button>
+                    <span flex="5"></span>
+                    <md-button onclick="window.history.go(-1); return false;;">Cancel</md-button>
```

Comment:
@nick22891 For the onclick handler, try to place the actions within a function in the controller instead of including it into the html code. We are trying to maintain separation of concerns.